tests/run.sh

```diff
@@ -80,9 +78,12 @@ function get_requirements() {
     #FIXME: The distributed binaries are compiled for OpenSSL 1.1
     #       As a workaround, we're installing the debian package (noble distribution).
     EXTRACT_DIR=$(mktemp -d mender-artifact.XXXXXX)
-    curl --fail \
-        "https://downloads.mender.io/repos/debian/pool/main/m/mender-artifact/mender-artifact_${MENDER_ARTIFACT_VERSION}-1%2bubuntu%2bnoble_amd64.deb" \
-        -o "$EXTRACT_DIR/mender-artifact.deb"
+    (
+        test -z "$MENDER_ARTIFACT_VERSION" && source ../.env
+        curl --fail \
+            "https://downloads.mender.io/repos/debian/pool/main/m/mender-artifact/mender-artifact_${MENDER_ARTIFACT_VERSION}-1%2bubuntu%2bnoble_amd64.deb" \
+            -o "$EXTRACT_DIR/mender-artifact.deb"
+    )
```

Comment:
Have to execute this in a sub-shell since the `source` will overwrite the entire environment, but the existing environment should take precedence.